src/components/Network/Network.vue

```diff
@@ -62,31 +82,39 @@
 <script>
 import TopBar from '@/components/TopBar'
 import ServiceModal from './ServiceModal'
+import IngressModal from './IngressModal'
 
 export default {
   name: 'Network',
-  components: { TopBar, ServiceModal },
+  components: { TopBar, ServiceModal, IngressModal },
   computed: {
     namespace () {
-      return this.$store.getters['users/get_user']['k8s_namespace_names'].join(', ')
+      return this.$store.getters['users/get_user']['namespace_names'].join(', ')
     },
     services_data () {
       return this.$store.getters['services/get_services']
     },
     ingresses_data () {
       return this.$store.getters['ingresses/get_ingresses']
+    },
+    check_available () {
```

Comment:
I don't get this function, locally on my machine its saying "Line 101 this.ingresses_data() is not a function.
And why are you returning an empty string?

Reply:
was wip